homeassistant/components/nhc2/light.py

```diff
@@ -0,0 +1,101 @@
+"""Support for NHC2 lights."""
+import logging
+from homeassistant.components.light import Light
+
+from .helpers import nhc2_entity_processor
+from nhc2_coco import CoCoLight, CoCo
+
+from .const import DOMAIN, KEY_GATEWAY, BRAND, LIGHT
+
+KEY_GATEWAY = KEY_GATEWAY
+KEY_ENTITY = 'nhc2_lights'
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Load NHC2 lights based on a config entry."""
+    hass.data.setdefault(KEY_ENTITY, {})[config_entry.entry_id] = []
+    gateway: CoCo = hass.data[KEY_GATEWAY][config_entry.entry_id]
```

Comment:
I don't think this is valid syntax. If it is however, what is it supposed to do?

Reply:
This is Python 3.6 typing syntax, we support a minimum of python 3.5.3

Reply:
Will look at those this weekend, thxç